code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -149,25 +149,42 @@
 	if(locate(/obj/structure/spacevine) in get_turf(src))//Heal if we are on vines
 		if(withering)
 			to_chat(src, span_notice(" The vines nourish you, healing your wounds."))
-		adjustHealth(-maxHealth*0.05)
+			stop_automated_movement = 0
+		adjustHealth(-maxHealth*0.1)
```

Comment:
```suggestion
		adjustHealth(-maxHealth*0.05)
```

I don't believe they should heal as fast as they do now that they have this much health having observed.